Arrays/matrixoperationusing2d.cpp

```diff
@@ -0,0 +1,231 @@
+//Operations on matrix using 2-D array
+//standred run should look like this
+/* For First Matrix :
+ enter row :2
+
+ enter column :2
+Enter element a11 : 1
+Enter element a12 : 2
+Enter element a21 : 3
+Enter element a22 : 4
+
+ For Second Matrix :
+ enter row :2
+
+ enter column :2
+Enter element a11 : 1
+Enter element a12 : 2
+Enter element a21 : 3
+Enter element a22 : 4
+
+
+ Matrix is: 
+1  2  
+3  4  
+
+
+ Matrix is: 
+1  2  
+3  4  
+
+Matrix of Sum of two matrix is: 
+
+
+ Matrix is: 
+2  4  
+6  8  
+
+Matrix of Substraction of two matrix is: 
+
+
+ Matrix is: 
+0  0  
+0  0  
+
+Matrix of Multiplication of two matrix is: 
+
+
+ Matrix is: 
+7  10  
+15  22  
+
+Transpose of matrix is: 
+
+
+ Matrix is: 
+1  3  
+2  4
+*/ 
+
+#include <iostream>
+
+using namespace std;
+
+class Matrix{
+    int row;
+    int column;
+    int m[5][5];
+    public:
+    Matrix(){
+      column=row=0;
+      int m[0][0];
+    };
+    Matrix(int,int);
+    Matrix(Matrix &x);
+    void displaydata(Matrix);
+    void add(Matrix,Matrix);
+    void sub(Matrix,Matrix);
+    void mul(Matrix, Matrix);
+    void trns(Matrix);
+    ~Matrix(){
```

Comment:
Can you please explain this class further?

Reply:
it is just used as a destructor